### ISSUE TITLE
account - request-limit-increase support for cloudformation

### ISSUE THREAD

**Issue by mkundrapu** [cloud-custodian/cloud-custodian#4279]
Team, As per custodian release notes for “0.8.28.0”, I see this new feature “account - request-limit-increase support for iam & cloudformation” added to custodian. So to utilize this, started creating a policy, but i see this validation error.

**resource:account 'CloudFormation' is not one of [u'EC2', u'ELB', u'VPC', u'AutoScaling', u'RDS', u'EBS', u'SES', u'IAM']**

I tried with latest version of c7n and also specific version (0.8.28.0) and see same error. Could some one look in to it?

**Comment by kapilt** [comment]
s/team/community.. pull requests welcome, we're looking at new service-quotas feature here.

**Comment by kumarsiyer** [comment]
Yeah this need to be replaced with https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/service-quotas.html

**Comment by kapilt** [comment]
#4251 covers service quota integration, I’ve got something in progress for it
